src/pfe/file-watcher/server/test/scripts/setup.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+# Set up default options value for test
+TEST_BRANCH="master"
+
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: [-<option letter> <option value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -f # Setup function, currently support 'install' and 'uninstall' - Mandatory
+    -h # Display the man page
+EOF
+}
+
+# Install function does the following things
+# 1 Build Codewind images
+# 2 Run Codewind containers
+function install {
+    ../../../../../run.sh
```

Comment:
Can we pass the `CW_DIR` variable to this file from exec.sh as well and use it here as well?

Reply:
We can't because there is no relationship between `setup.sh` and `exec.sh`, `test.sh` calls `setup.sh` and `exec.sh`. Since we don't know the path of user's Codewind repo, we can only get the Codewind repo path using the relative path. If we want the absolute path we can do something like this
```
cd ../../../../../
CW_DIR=$(pwd)
cd -
```
But again, it's still get it from relative path